src/main/java/org/gridsuite/ds/server/dto/dynamicmapping/automata/BasicProperty.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.ds.server.dto.dynamicmapping.automata;
+
+import org.gridsuite.ds.server.utils.PropertyType;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
```

Comment:
Use @Schema(description = "blabla") as you did it for other ones ?

Reply:
DONE in https://github.com/gridsuite/dynamic-simulation-server/pull/95/commits/9c1f3283878c39cac8535ed7157cc5e04ff81e14

Reply:
DONE